tools/label-api.sh

```diff
@@ -40,8 +39,8 @@ exit_with_usage () {
 execute () {
   curl -H "Authorization: token $GITHUB_TOKEN" \
       --request POST \
-      --data "{\"event_type\": \"label-api-snapshot\", \"client_payload\": { \"label\": \"${LABEL}\", \"snapshot_id\": \"${SNAPSHOT_ID}\" } }" \
-      https://api.github.com/repos/covid-projections/covid-data-model/dispatches
+      --data "{\"inputs\": { \"snapshot_id\": \"${SNAPSHOT_ID}\" } }" \
```

Comment:
I think you have to specify the ref as well? Maybe that's changed, but i think i remember that was the case (and i think is the pattern elsewhere)

Reply:
I'm not sure!  I was hoping it was optional and removed it.  I think I'll try it and find out. 😁

Reply:
such is life with github actions!

Reply:
You were right! 😆 

```
{
  "message": "Invalid request.\n\n\"ref\" wasn't supplied.",
  "documentation_url": "https://docs.github.com/rest/reference/actions#create-a-workflow-dispatch-event"
}
```

Fixed here: https://github.com/covid-projections/covid-data-model/commit/345aa948a67bee005b4e805e0fd3f14e22f43299